src/components/company/chart.js

```diff
@@ -86,7 +86,6 @@ class ChartContainer extends React.Component {
     }
 }
 
-
 const DisplayChart = ({ state }) => {
```

Comment:
why not just pass history in. The function should know as much as it needs to and no more.